pypose/function/geometry.py

```diff
@@ -126,8 +126,16 @@ def reprojerr(points, pixels, intrinsics, extrinsics=None):
             The shape has to be (..., 3, 3).
         extrinsics (``LieTensor``, optional): The camera extrinsics.
             The shape has to be (..., 7). Default: ``None``.
+        reduction (``str``, optional): The reduction to apply on the output: ``'none'``
+            | ``'sum'`` | ``'norm'``
+            ``'none'``: No reduction is applied
+            ``'sum'``: The reprojection error on each component (u, v) is summed for
+            each pixel (L1 Norm)
+            ``'norm'``: The reprojection error's L2 norm for each pixel
```

Comment:
It seems that you need commas for every line, otherwise, those lines are connected automatically by docstring. See this auto-generated doc for your PR:
https://pypose--260.org.readthedocs.build/en/260/generated/pypose.reprojerr.html

Reply:
You may generate docs locally following this [readme](https://github.com/pypose/pypose/blob/main/CONTRIBUTING.md).